v2.2/debug-merge-logs.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Merge debug logs for all nodes
+summary: Learn the command for merging the collected debug logs from all nodes in your cluster.
+toc: true
+---
+
+The `debug merge-logs` [command](cockroach-commands.html) merges the logs for all nodes in the cluster collected using the [`debug zip`](https://www.cockroachlabs.com/docs/stable/debug-zip.html) command. This command helps in debugging cluster issues by making it easier to parse the logs for all nodes and locating an issue.
+
```

Comment:
Maybe it's worth noting that in addition to merging logs into a single stream, `debug merge-logs` also enables selecting a specific time interval using the `--from` and `--to` flags as well as filtering messages based on a regular expression pattern using the `--filter` flag. The filter flag can be handy because unlike regular grep, it is aware of multi-line log messages.